### ISSUE TITLE
add Support for Java Stub in JUnitFragment

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-core#1482]
add Support for Java Stub in JUnitFragment

if generateXtendStub = false a java stub should be created
if generateStub = false, nothing should be created

**Comment by cdietrich** [comment]
@nbhusare is working on this

**Comment by nbhusare** [comment]
> if generateStub = false, nothing should be created

@cdietrich Sorry, what do you mean by **nothing**? In the current state, setting `generateStub = false` only generates the `DslInjectorProvider` class, no stub for the unit test is generated.

**Comment by cdietrich** [comment]
noting = no stub

**Comment by nbhusare** [comment]
@cdietrich Tx for clarifying. Just setting `generateStub = true` generates Junit stub in Xtend. Do you want that to be changed to Java?

**Comment by cdietrich** [comment]
no

generateStub = true
generateXtendStub = false

should give java.

(this should be backwards compatible to current state

**Comment by cdietrich** [comment]
(needs to be tested)

**Comment by cdietrich** [comment]
i had a look. the current impl really is bad

```
		if (isGenerateStub)
			generateExampleRuntimeTest.writeTo(projectConfig.runtimeTest.src)
```

thus we would have to change generateXtendStub to be true by default to stay compatible

**Comment by cdietrich** [comment]
Fixed in 2.22
